configs/defdockerfiles/alpine

```diff
@@ -0,0 +1,49 @@
+# Docker image for "edge-orchestration"
+### alpine:3.12
+ARG PLATFORM
+FROM $PLATFORM/alpine:3.12
```

Comment:
How about using latest version(3.14)?

Reply:
I tried to use the latest version of the Alpine, but during the build for ARM architecture, errors associated with `syscalls` arise. Need a deep analysis of the `qemu` since I think that this is a problem with it (need to test with the qemu latest version)